msteams-platform/index-test.yml

```
@@ -0,0 +1,171 @@
+### YamlMime:Hub
+
+title: Microsoft Teams developer documentation # < 60 chars
+# brand: aspnet | azure | dotnet | dynamics | m365 | ms-graph | office | power-bi | power-platform | sql | sql-server | vs | visual-studio | windows | xamarin
+summary: "Get documentation, sample code, tutorials, and more to help you build custom Microsoft Teams apps."
+brand: teams
+
+metadata:
+  title: Microsoft Teams Platform developer documentation # Required; page title displayed in search results. Include the brand. < 60 chars.
+  description: The Microsoft Teams developer platform makes it easy for developers to integrate their own apps and services to improve productivity, make decisions faster, provide focus (by reducing context switching), and create collaboration around existing content and workflows. . # Required; article description that is displayed in search results. < 160 chars.
+  ms.service: msteams #Required; service per approved list. service slug assigned to your service by ACOM.
+  ms.topic: hub-page # Required
+  ms.collection: collection # Optional; Remove if no collection is used.
+  author: laujan #Required; your GitHub user alias, with correct capitalization.
+  ms.author: lajanuar #Required; microsoft alias of author; optional team alias.
+  ms.date: 05/29/2020 #Required; mm/dd/yyyy format.
+
+
+highlightedContent:
+# itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | tutorial | video | whats-new
+   items: 
+    - title: What is Teams!
+      itemType: video
+      url: https://www.microsoft.com/en-us/videoplayer/embed/RE4Jqbg
+    - title: New to Microsoft Teams development? 🚀Start here! 
+      itemType: get-started
+      url: /microsoftteams/platform/overview
+    - title: Plan your app
+      itemType: overview
+      url: /microsoftteams/platform/concepts/design/understand-use-cases
+    - title: Design your app
+      itemType: overview
+      url: /microsoftteams/platform/concepts/design/design-teams-app-overview
+    - title: Extend your app with Microsoft Graph
+      itemType: overview
+      url:  /graph/teams-concept-overview
+    - title: Deploy and publish your app
+      itemType: overview
+      url: /microsoftteams/platform/concepts/deploy-and-publish/appsource/publish
+    - title: Low-code apps, bots, flows, and reports using Microsoft Power Platform
```

```suggestion
    - title: Create low-code apps, bots, and flows with Microsoft Power Platform
```
Added a verb for parallel structure.